### PR TITLE
Support netstandard1.0

### LINKED ISSUE #300: Support .NET Standard 1.1 (including .NET Core 1.0 onwards)
I'm willing to take a stab.

### PR DESCRIPTION
**Pull request by adamralph**
fixes #300

### CODE REVIEW

File: Xbehave.sln [hunk 1]
```diff
@@ -43,36 +37,43 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
...
 		Dictionary.xml = Dictionary.xml
+		global.json = global.json
 		LICENSE = LICENSE
 		README.md = README.md
 		stylecop.json = stylecop.json
 	EndProjectSection
 EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Xbehave.Core", "src\Xbehave.Core\Xbehave.Core.xproj", "{37910B4A-996D-4DB1-816E-7ABAAE43CEC2}"
+EndProject
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Xbehave.Execution", "src\Xbehave.Execution\Xbehave.Execution.xproj", "{27AB17E5-1796-4DBD-8FFE-99249114C93A}"
```
Comment (blairconrad) [inline Xbehave.sln:48]: I never in a million years would've figured out the new name. Even looking at it, I'm not sure why.

Comment (adamralph) [reply Xbehave.sln:48]: This is the key part https://github.com/xbehave/xbehave.net/pull/321/commits/b1eb2dffccb862c19b1152ac77ccaf0395fc812a#diff-652202c06576bdabd2ef440e63769c32R20

xunit uses that to figure out which TFN the execution lib supports.

Comment (adamralph) [reply Xbehave.sln:48]: Together with this.... https://github.com/adamralph/xbehave.net/blob/b1eb2dffccb862c19b1152ac77ccaf0395fc812a/src/Xbehave.Execution/Properties/AssemblyInfo.cs#L10

Comment (blairconrad) [reply Xbehave.sln:48]: Ah. Thanks. I'm not sure I could reproduce it, but right now, I think I see.

**Pull request merged**